### PR TITLE
added image thumbnail with loader for slow connections

### PR DESCRIPTION
**Pull request by marcosparagna-eb**

### CODE REVIEW

File: src/components/styles.js [hunk 1]
```diff
@@ -30,6 +30,25 @@ export const globalSyles = {
   },
 };
 
+export const thumbnail = css({
```
Comment (kwelch) [inline src/components/styles.js]: If these are specific to a component can you add them to the component file?

Comment (marcosparagna-eb) [reply src/components/styles.js]: absolutely, didn't know what the logic was there!

**Comment by marcosparagna-eb** [comment]
adding screenshot for reference!
disclaimer: not a designer 🍡 

image

**Comment by kwelch** [comment]
I have deployment previews so i can see it here: https://deploy-preview-11--flamboyant-liskov-561e43.netlify.com/

**Pull request merged**